GeomorphicFloodIndex/doGeomorphicFloodIndex.py

```diff
@@ -478,8 +478,8 @@ def accept(self):
             debug=0
             calibration=0
 
-            if self.checkBoxDebug. isChecked():
-                debug=1
+            #if self.checkBoxDebug. isChecked():
+            #    debug=1
             if self.checkBoxCalibration. isChecked():
                 calibration=1
```

Comment:
```suggestion
            if self.checkBoxDebug.isChecked():
                debug=1
            if self.checkBoxCalibration.isChecked():
                calibration=1
```

Thanks @caffarel. Why the need to comment out such lines?

Reply:
I didn't work for us while these were there but you can put them back if they are important